resources/wf_builder.rb

```diff
@@ -69,10 +69,22 @@
       command "knife ssl fetch -s https://#{server} -c #{Chef::Config['config_file']}"
       not_if "knife ssl check -s https://#{server} -c #{Chef::Config['config_file']}"
       ignore_failure true
+    end unless server.nil? || server.empty?
+  end
+
+  ruby_block 'configure SSL certificates' do
+    block do
+      @cacert_pem = ::File.read('/opt/chefdk/embedded/ssl/certs/cacert.pem')
+      ::Dir.glob('/etc/chef/trusted_certs/*.crt').each do |crt|
+        c = ::File.read(crt)
+        @cacert_pem << "\n" << c unless @cacert_pem.include?(c)
+      end
     end
   end
 
-  execute 'cat /etc/chef/trusted_certs/*.crt >> /opt/chefdk/embedded/ssl/certs/cacert.pem'
+  file '/opt/chefdk/embedded/ssl/certs/cacert.pem' do
+    content @cacert_pem
```

Comment:
Instead of lazy we should not use the instance variable syntax since this won't be used as an accessor.

Preferably set `cacert_pem = nil` above the ruby block and reference it by it's name.